### PR TITLE
feat(model): Add optional message field to auditstamp

### PR DESCRIPTION
**Pull request by dexter-mh-lee**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by jjoyce0510 — APPROVED** [review]

File: li-utils/src/main/pegasus/com/linkedin/common/AuditStamp.pdl [hunk 1]
```diff
@@ -19,4 +19,9 @@ record AuditStamp {
    * The entity (e.g. a service URN) which performs the change on behalf of the Actor and must be authorized to act as the Actor.
    */
   impersonator: optional Urn
+
+  /**
+   * Additional message to keep track of in the event
```
Comment (shirshanka) [inline li-utils/src/main/pegasus/com/linkedin/common/AuditStamp.pdl:24]: what are the sort of things that we would expect to be written into this `message` field for AuditStamp?

Comment (shirshanka) [reply li-utils/src/main/pegasus/com/linkedin/common/AuditStamp.pdl:24]: The doc string is not super clear about what the purpose of this field is

Comment (jjoyce0510) [reply li-utils/src/main/pegasus/com/linkedin/common/AuditStamp.pdl:24]: Agreed

**Comment by jjoyce0510** [comment]
Let's wait til we release new CLI version - this has potential to break the old actions framework because its not forwards compatible

**Comment by gabe-lyons** [comment]
closing in favor of https://github.com/datahub-project/datahub/pull/5611

**Pull request closed without merging**